<launch>
  <!-- True for simulation, false for real-time and bagfiles -->
  <param name="/use_sim_time" value="false"/>

      <!-- <node pkg="tf" type="static_transform_publisher" name="imu_to_odom"
                args="0 0 0 -3.14159 0 0 /odom /imu_link 100" /> -->
     <!--<node pkg="tf2_ros" type="static_transform_publisher" name="imu_transform" args="0 0 0 0 0 3.14159 base_link imu_link" /> -->

  <group ns="core_nav">
    <node pkg="core_nav"
          name="core_nav"
          type="core_nav_node"
          output="screen">

      <!-- Initial pose -->
      <rosparam param="init">
        position: {x: 0.691949695095547, y: -1.395750757746275, z: 312.2643980319930}
        velocity: {vx: 0.0, vy: 0.0, vz: 0.0}
        orientation: {qx: 0.0, qy: 0.0, qz: 0.0, qw: 1.0}

        <!-- orientation: {qx: 0.737277336810124, qy: 0.0, qz: 0.0, qw: 0.675590207615660} -->
        <!--position_sigma: {x: 1.0, y: 1.0, z: 1.0}
        velocity_sigma: {vx: 0.1, vy: 0.1, vz: 0.1}
        orientation_sigma: {roll: 0.02, pitch: 0.02, yaw: 0.02}-->
      </rosparam>

      <!-- Rates -->
      <rosparam file="$(find core_nav)/config/parameters.yaml"/>
      <rosparam file="$(find core_nav)/config/init_params.yaml"/>
    </node>
  </group>
</launch>
